src/components/DrugChartSlider/DrugChartSlider.scss

```diff
@@ -0,0 +1,176 @@
+.dose-field-with-units {
+  display: flex;
+  flex-direction: row;
+  width: 55%;
+  justify-content: flex-start;
+  align-items: flex-end;
+  margin-right: 10px;
+
+  .dropdown {
+    width: 142.449px;
+  }
+}
+
+.duration-field-with-units {
+  display: flex;
+  flex-direction: row;
+  width: 55%;
+  justify-content: flex-start;
+  align-items: flex-end;
+  margin-right: 10px;
+
+  .dropdown {
+    width: 142.449px;
+  }
+}
+
+.-TaKcdG2c0NUzkYaFWOzlQ\=\= {
+  width: 142.449px;
+  margin-right: 5px;
+}
+
+.frequency .dropdown {
+  margin-right: 0px !important;
+}
+
+.date-picker-carbon {
+  width: 45%;
+  .bx--date-picker .bx--date-picker--single,
+  .bx--date-picker-container {
+    width: 100%;
+  }
+}
+
+.schedule-time {
+  margin-right: 5px;
+}
+
+.schedule-section,
+.frequency,
+.start-time,
+.inline-field,
+.notes-sections,
+.instructions,
+.additional-instructions {
+  padding: 10px 0 10px 0;
+}
+
+.schedule-section,
+.inline-field {
+  display: flex;
+  align-items: inherit;
+}
+
+.schedule-section {
+  flex-direction: column;
+  flex-wrap: wrap;
+  .inline-field {
+    padding: 0;
+  }
+}
+
+.route {
+  width: 45%;
+  margin-right: 0px;
+
+  .dropdown {
+    width: 100%;
+  }
+}
+
+.time-error {
+  margin: 0;
+  margin-top: 4px;
+  font-size: 0.75rem;
+  color: #da1e28;
+}
+
+.time-warning {
+  margin: 0;
+  margin-top: 4px;
+  font-size: 0.75rem;
+  color: #2b95ff;
+}
+
+.bx--text-input-wrapper .bx--text-input {
+  width: 100%;
+  overflow: auto;
+  border: none;
+  background-color: #efefef;
+  color: #8d8d8d;
+}
+
+button[class*="bx--"]:disabled {
+  opacity: 1;
+  border: none;
+  border-radius: 0px;
+}
+
+.bx--text-input__readonly-icon {
+  display: none;
+  cursor: not-allowed;
+}
+
+.bx--date-picker.bx--date-picker--single .bx--date-picker__input {
+  background: #efefef;
+  width: 100% !important;
+}
+
+.bx--date-picker.bx--date-picker--single {
+  width: 100%;
+}
+
+.bx--date-picker__input:disabled,
+.bx--number input[type="number"]:disabled {
+  border: none;
+}
+
+.bx--number input[type="number"]:disabled {
+  font-size: 0.85rem;
+  font-weight: 400;
+  color: #8d8d8d;
+  cursor: not-allowed;
+  background-color: #efefef;
+}
+
+.bx--number__control-btn:disabled {
```

Comment:
are these carbon UI over ridden styles only needed for this DrugChartSlider or is it generic at every place where ever we gonna use? If it's specific to only DrugCartSlider then I would suggest to wrap it inside a unique parent class so that it would impact other places and if it is applicable to every place then move it to common carbon style. You can also identify which are generic which can be moved